os/drivers/lcd/mipi_lcd.c

```diff
@@ -129,6 +129,8 @@ struct mipi_lcd_dev_s {
 
```

Comment:
you moved to get each lcd header to {lcd ic chip}.c

Please move include theae
```
#if defined(CONFIG_LCD_ST7785)
#include <tinyara/lcd/st7785.h>
#elif defined(CONFIG_LCD_ST7701)
#include <tinyara/lcd/st7701.h>
#elif defined(CONFIG_LCD_ST7701SN)
#include <tinyara/lcd/st7701sn.h>
```